pkg/controllers/management/auth/user.go

```diff
@@ -164,6 +171,39 @@ func (l *userLifecycle) Updated(user *v3.User) (runtime.Object, error) {
 	if err != nil {
 		return nil, err
 	}
+
+	if user.Enabled == nil || !*user.Enabled {
```

Comment:
`user.Enabled == nil` means the user is disabled? Isn't it the default value true like token.Enabled? In that case we should not delete tokens if `user.Enabled == nil`

Reply:
Fixed with commit 73501deeb6e905088f0eab7040de8c26f348d4b9 (rebase!)